src/uimports.cpp

```diff
@@ -90,8 +90,11 @@ static BOOL UPDATE_IMPORTS_XX(HANDLE hProcess,
         }
 
         DETOUR_TRACE(("ish[%d] : va=%08x sr=%d\n", i, ish.VirtualAddress, ish.SizeOfRawData));
-
-        // If the file didn't have an IAT_DIRECTORY, we assign it...
+        
+        // If the linker didn't suggest and IAT in the data directories, the loader will look for the
+        // section of the import directory to be used for this instead. Since we put out new IMPORT_DIRECTORY
```

Comment:
nit: 
Would you mind reformating the comments so they obey the 80 character line limit convention? 
It seems to be obeyed in majority of the detours code base, we should try to keep things consistent.

Reply:
I pushed a fix for this